### ISSUE TITLE
Fix TypeError with benchmark-rule script

### ISSUE THREAD

**Issue by ybiquitous** [stylelint/stylelint#3804]
> Clearly describe the bug

When running the command in [the documentation](https://github.com/stylelint/stylelint/blob/f865f3df5d67f97aa2d694d56f81d1ba5613f8b7/docs/developer-guide/rules.md#improving-the-performance-of-a-new-or-an-existing-rule), the following error is raised:

```
$ npm run benchmark-rule -- selector-combinator-space-after never

> stylelint@9.8.0 benchmark-rule /Users/koba/git/stylelint/stylelint
> node scripts/benchmark-rule.js "selector-combinator-space-after" "never"

/Users/koba/git/stylelint/stylelint/scripts/benchmark-rule.js:34
const rule = rules[ruleName].apply(null, normalizeRuleSettings(parsedOptions));
                             ^

TypeError: Cannot read property 'apply' of undefined
    at Object.<anonymous> (/Users/koba/git/stylelint/stylelint/scripts/benchmark-rule.js:34:30)
    at Module._compile (internal/modules/cjs/loader.js:707:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:718:10)
    at Module.load (internal/modules/cjs/loader.js:605:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:544:12)
    at Function.Module._load (internal/modules/cjs/loader.js:536:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:760:12)
    at startup (internal/bootstrap/node.js:308:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:878:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! stylelint@9.8.0 benchmark-rule: `node scripts/benchmark-rule.js "selector-combinator-space-after" "never"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the stylelint@9.8.0 benchmark-rule script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/koba/.npm/_logs/2018-11-12T15_37_35_699Z-debug.log
```

> Which rule, if any, is the bug related to?

There may be something wrong in `scripts/benchmark-rule.js`.

https://github.com/stylelint/stylelint/blob/f865f3df5d67f97aa2d694d56f81d1ba5613f8b7/scripts/benchmark-rule.js#L34

> What CSS is needed to reproduce the bug?

N/A

> What stylelint configuration is needed to reproduce the bug?

N/A

> Which version of stylelint are you using?

`9.8.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI script for developers.

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

N/A

> What did you expect to happen?

- Benchmark for all existing rules succeed.
- All examples of the benchmark script in documentation are fixed.

> What actually happened (e.g. what warnings or errors did you get)?

`npm run benchmark-rule` command failed.
